test_llm_optimum.py

```diff
@@ -0,0 +1,96 @@
+import argparse
```

Comment:
I added that in a commit with a revertme in the title: it will be removed later and was just here as an illustration of how the instantiation happens.

I now realize that you are more used to review files all at once than reviewing commits one by one, so this went under your radar and michael's.